### PR TITLE
Softone BI (v1.0.0.0) connector documentation.

### PR DESCRIPTION
**Pull request by basil-rormose**
Hi,

submitting the documentation for the SoftOne BI connector as part of completing the certification process.

### CODE REVIEW

File: powerquery-docs/Connectors/SoftoneBI.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+---
+title: Softone BI connector
```
Comment (bezhan-msft) [inline powerquery-docs/Connectors/SoftoneBI.md:2]: Should this be "SoftOne BI" to be consistent with your connector Publish

File: powerquery-docs/Connectors/SoftoneBI.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+The SoftOne BI connector is supported from Soft1 Series 5 version 500.521.11424 or later and Atlantis ERP version 3.3.2697.1 or later.
+
+## Capabilities supported
+
+* Import
+
+## Connection instructions 
+
+Softone provides a number of templates as Power BI template files (.pbit) (e.g., Sales & Collections, Finance) that you can use as is or customize which will provide you with a great start to your BI project.
```
Comment (bezhan-msft) [inline powerquery-docs/Connectors/SoftoneBI.md:34]: nit: Softone -> SoftOne

**Review by bezhan-msft — APPROVED** [review]
Looks good overall with minor changes. @basil-rormose, we will make the minor changes on the Microsoft side including small updates to adhere with Microsoft style guides.

File: powerquery-docs/Connectors/SoftoneBI.md (hunk 1 above)
Comment (basil-rormose) [reply powerquery-docs/Connectors/SoftoneBI.md:2]: Yes, that is correct.

**Pull request merged**